packages/notifi-frontend-client/lib/client/clientFactory.ts

```diff
@@ -32,5 +34,6 @@ export const newFrontendClient = (args: ConfigFactoryInput) => {
   const config = newFrontendConfig(args);
   const service = newNotifiService(config);
   const storage = newNotifiStorage(config);
-  return new NotifiFrontendClient(config, service, storage);
+  const wsurl = envWSGraphqlUrl(config.env);
+  return new NotifiFrontendClient(config, service, storage, wsurl);
```

Comment:
Similar comment as above. 
If we migrate all the gql related operations to `notifi-graphql` package, we will no longer need to have input the wsrul here I believe. How do you think